jerry-core/ecma/operations/ecma-lex-env.c

```diff
@@ -270,19 +270,21 @@ ecma_op_get_binding_value (ecma_object_t *lex_env_p, /**< lexical environment */
 
     ecma_object_t *binding_obj_p = ecma_get_lex_env_binding_object (lex_env_p);
 
-    if (ecma_op_object_get_property (binding_obj_p, name_p) == NULL)
+    ecma_value_t result = ecma_op_object_find (binding_obj_p, name_p);
+
+    if (!ecma_is_value_found (result))
```

Comment:
I mean if it is not an ECMA_SIMPLE_VALUE_NOT_FOUND then it might be anything else. There is not guarantee that is a simple value. What about adding an assert to `ecma_is_value_found` to check the parameter is always a simple value?